### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by CCFenner**
- move test files to `test` folder
- move image content to `src` folder
- cleanup ignores

### CODE REVIEW

File: test/runTests.sh [hunk 1]
```diff
@@ -1,15 +1,15 @@
 #/bin/bash
 
 export BUILD_ARTIFACT=dummy.mtar
-export NODE_TEST_FOLDER=test/node
-export MAVEN_TEST_FOLDER=test/maven
+export NODE_TEST_FOLDER=node
+export MAVEN_TEST_FOLDER=maven
 
-docker build -t mta-archive-builder . || { echo "[ERROR] Failed to build mta-archive-builder image."; exit 1; }
+docker build -t mta-archive-builder $PWD/.. || { echo "[ERROR] Failed to build mta-archive-builder image."; exit 1; }
```
Comment (fwilhe) [inline test/runTests.sh]: this will break with whitespaces?

**Comment by fwilhe** [comment]
Build error in travis:

```
unable to prepare context: unable to evaluate symlinks in Dockerfile path: lstat /home/travis/build/SAP/devops-docker-mta-archive-builder/test/integration/Dockerfile: no such file or directory
The command "cd test/integration/ && chmod +x runTests.sh && ./runTests.sh" exited with 1.
```

**Comment by fwilhe** [comment]
Looks good on a first glance, but I’m hesitant to merge such non-trivial PRs before our travis builds are green again. Have not yet had a deep look.

**Comment by CCFenner** [comment]
@fwilhe build is green again

**Comment by fwilhe** [comment]
Ok it took me a while to understand that ‘only’ integration is run as of now. But that is fine, was not better before.

**Pull request merged**